### DISCUSSION TITLE
Verona Components Missing

### DISCUSSION THREAD

**Discussion by Bubily77** [primefaces/community#2571] [Figma UI Kit]
We recently purchased PrimeOne 2.2. along with the Verona template. I'm wrapping my head around how exactly are we supposed to use the Verona template in Figma. I can see there are tokens and styles available, but the only components we have are the top and sidebar. Nothing else. Meaning if I want to design using Verona's color palette or fonts, I have to use a component from the PrimeOne kit and then manually either strip off the applied tokens and apply the color/text styles from the Verona file or I have to manually update the existing PrimeOne tokens with the values from the Verona template.

Am I missing something or is this how it's supposed to work? How exactly do we design using the Verona Figma file? 

Cheers

**Comment by onursenture** [comment]
Hello,

The Figma files that come with the templates are not designed to assist in your development processes, but rather they are intended to be implemented by PrimeTek engineers for these premium templates. Therefore, it is not really possible for them to play a supportive role in your development processes as you mentioned.

Additionally, PrimeOne UI Kit is not connected to the premium templates. The PrimeOne UI Kit currently only supports the Aura preset (formerly known as the theme).

In summary, the Figma support for our premium templates is limited to the Figma file you have, and there is currently no Figma solution that includes all the components used in the premium template.
